app/assets/stylesheets/_users.scss

```diff
@@ -0,0 +1,135 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+body {
+  font-family: candara;
+}
+
+$light-grey: lighten(#b2beb5, 10%);
+
+form {
+  border-right: 2px solid $light-grey;
+}
+
+.row.container {
+  box-shadow: none;
+  width: 60%;
+  margin: auto;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+}
+
+h2 {
+  text-align: center;
+  font-size: 35px;
+}
+
+.row.entry {
+  height: 35px;
+  width: 450px;
+}
+
+.row.submit {
+  right: 48.1955px;
+  position: relative;
+  top: 20px;
+}
+
+.container.col.l4 {
+  text-align: right;
+  font-family: candara;
+}
+
+.container.col.l8 {
+  text-align: center;
+  padding: 0;
+}
+
+.row.signup {
+  font-family: candara;
+  text-align: right;
+}
+
+.btn.waves-effect.waves-light {
+  height: 40px;
+  width: 150px;
+  vertical-align: center;
+}
+
+
+.fa.fa-google-plus {
+  padding-right: 5px;
+  position: absolute;
+  left: 10px;
+  height: 30px;
+}
+
+.fa.fa-facebook {
+  padding-right: 5px;
+  position: absolute;
+  left: 10px;
+  height: 30px;
+}
+
+.fa.fa-twitter {
+  padding-right: 5px;
+  position: absolute;
+  left: 10px;
+  height: 30px;
+}
+
+.btn.waves-effect.waves-light.blue {
+  width: 200px;
+  text-align: center;
+  background-color: #338574 !important;
+}
+
+.btn.waves-effect.waves-light.facebook.blue {
+  background-color: #3b5998 !important;
+  width: 150px !important;
+}
+.btn.waves-effect.waves-light.twitter.blue {
+  background-color: #4099ff !important;
+  width: 150px !important;
+}
+
+.btn.waves-effect.waves-light.oauth.red {
+  background-color:  #d34836 !important;
+}
+
+.fa-word {
+  margin-left: 10px;
+}
+
+.row.signup p {
+  text-align: right;
+  padding-right: 34px;
+  margin-top: 10px;
+  font-size: 1.2em;
+}
+
+.container.col.l8 p {
+  margin-top: 40px;
+  position: relative;
+  right: 48.1955px;
+}
+
+
+.row.oauth {
+  padding-right: 22.5px;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.bar {
+  font-size: 20px;
+  font-weight: 100;
+  position: absolute;
```

Comment:
, ,,,, ,,,                                                                                                                                                                                                                                                                                                                                         Sent from my BlackBerry 10 smartphone.                                                                                                                                                                                                                From: HoundSent: Wednesday, 2 December 2015 15:04To: andela/getmyshopReply To: andela/getmyshopSubject: Re: [getmyshop] Feature: Signup page (#3)In app/assets/stylesheets/_users.scss:
> +  right: 48.1955px;
> +}
> +
> +
> +.row.oauth {
> +  padding-right: 22.5px;
> +}
> +
> +a:hover {
> +  text-decoration: underline;
> +}
> +
> +.bar {
> +  font-size: 20px;
> +  font-weight: 100;
> +  position: absolute;

Properties should be ordered font-size, font-weight, left, position, top

—Reply to this email directly or view it on GitHub.